azkaban-common/src/main/java/azkaban/executor/container/ContainerCleanupManager.java

```diff
@@ -202,10 +216,8 @@ private void retryFlowQuietly(ExecutableFlow flow, Status originalStatus) {
   private void deleteContainerQuietly(final int executionId) {
     try {
       this.containerizedImpl.deleteContainer(executionId);
-    } catch (final ExecutorManagerException eme) {
-      logger.error("ExecutorManagerException while deleting container.", eme);
-    } catch (final RuntimeException re) {
-      logger.error("Unexpected RuntimeException while deleting container.", re);
+    } catch (final Exception e) {
+      logger.error("Unexpected RuntimeException while deleting container.", e);
```

Comment:
can we adjust the message to log `Unexpected exception` instead of `Unexpected RuntimeException`?

Reply:
Changed the log message as requested.

Reply:
Thank you! Looks like `RuntimeException` is still there, perhaps the change was not `git add`?